app/controllers/spree/shipwire_webhook_controller.rb

```diff
@@ -0,0 +1,34 @@
+class Spree::ShipwireWebhookController < ActionController::Base
+  respond_to :json
+
+  before_action :validate_key, except: :subscribe
```

Comment:
this is done except `subscribe` but  `subscribe` seems to be the only action here. Am I missing something?